app/retail/templates/leaderboard.html

```diff
@@ -3,6 +3,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+  <script src="{% static "v2/js/lib/jquery.js" %}"></script>
```

Comment:
wouldn't jquery be loaded twice in this page in this case ?

Reply:
OK FIXED